share/smack/top.py

```diff
@@ -304,6 +305,7 @@ def default_clang_compile_command(args, lib = False):
   if args.memory_safety: cmd += ['-DMEMORY_SAFETY']
   if args.signed_integer_overflow: cmd += (['-ftrapv'] if not lib else ['-DSIGNED_INTEGER_OVERFLOW_CHECK'])
   if args.float: cmd += ['-DFLOAT_ENABLED']
+  if args.strings: cmd += ['-DSTRINGS_ENABLED']
```

Comment:
This is not the right way to turn on strings. Take a look at: https://github.com/smackers/smack/blob/develop/share/smack/top.py#L317